tsconfig.json

```diff
@@ -49,9 +49,11 @@
     }
   },
   "exclude": [
-    "e2e",
-    "./cypress.config.ts",
-    "**/*.cy.tsx"
+      "e2e",
+      "./cypress.config.ts",
+      "**/*.cy.tsx",
+      "src/app/core/basic-datatypes/string/edit-text/display-long-text/display-longtext.component.spec.ts",
+      "src/app/core/basic-datatypes/string/edit-text/display-long-text/display-unit.stories.ts"
```

Comment:
please revert these changes. You shouldn't have to change the central tsconfig settings for the project.